great_expectations/util.py

```diff
@@ -1668,7 +1669,7 @@ def convert_ndarray_decimal_to_float_dtype(data: np.ndarray) -> np.ndarray:
 
 
 def get_context(
-    project_config: Optional[Union["DataContextConfig", dict]] = None,
+    project_config: Optional[Union["DataContextConfig", Mapping]] = None,
```

Comment:
Could we fix up this `DataContextConfig` type annotation by using `from __future__ import annotations` and `if TYPE_CHECKING`?

Reply:
Good call :) doing now